src/utils/server-utils.ts

```diff
@@ -1,9 +1,14 @@
 import moment from 'moment';
 import { xml2js } from 'xml-js';
-import { merge, get, isString, union, isEmpty } from 'lodash';
-
+import { get, isEmpty, isString, merge, union } from 'lodash';
 import config from '../config/prism.json';
-import { AvailableDates } from '../config/types';
+import { AvailableDates, GroundstationLayerProps } from '../config/types';
+import { LayerDefinitions } from '../config/utils';
+import { GroundstationLayerData } from '../context/layers/groundstation';
+
+// Note: PRISM's date picker only works with dates where their time is midnight in the UTC timezone (possible dates passed in should be 2020-xx-xx 00:00 Midnight in UTC    -    2020-xx-xx 10:00 in Australia East Coast)
```

Comment:
The `midnight` here shouldn't matter. But the UTC should. The short version should be: Dates should be processed in UTC as soon as possible.